src/sass/components/_post-box.scss

```diff
@@ -1,54 +1,81 @@
-.post-box {
-  margin-bottom: 2rem;
-
-  .cover {
-    padding: 70% 25px 0 25px;
-    background: {
-      color: #f4f5f7;
-      size: cover;
-      position: center;
-    }
+.post-box-container {
+  display: block;
 
-    .details {
-      padding: 15px 20px;
-      display: flex;
-      background-color: #ffffff;
-      font-size: 14px;
+  .post-box {
+    margin-bottom: 2rem;
 
-      & > * {
-        flex: 1 1 auto;
+    .cover {
+      padding: 70% 25px 0 25px;
+      background: {
+        color: #f4f5f7;
+        size: cover;
+        position: center;
       }
 
-      .date {
-        text-align: left;
-      }
+      .details {
+        padding: 15px 20px;
+        display: flex;
+        background-color: #ffffff;
```

Comment:
zapewne mamy już zmienną z takim kolorem, moze warto jej tu użyć?

Reply:
`$post-box-bg: #fff;`

**To samo:** mogę nazwać *$bg-white*, *$post-box-bg* - odnoszące się do danego elementu, jeśli będziemy chcieli zmienić dany element (kolor) to w jednej zmiennej. Musiałem, stworzyć *#fff*